README.md

```diff
@@ -27,8 +27,24 @@ cp serverless.example.yml serverless.yml
 
 Test your potential deployment:
 
+The best way to test the Lambdas is to deploy them to your personal AWS account and then call them using the default
+URLs generated by AWS after deployment.
+
+You can also test them locally, but this is not the ultimate indicator of success.
+
+To deploy to AWS,
+[ensure your AWS credentials are accessible](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html)
+and then run:
+
 ```shell
-rm -rf .build .serverless;
+rm -rf .build .serverless; # Sometimes serverless caches build artifacts that can confound testing
+yarn sls deploy
+```
+
+To test locally:
+
+```shell
+rm -rf .build .serverless; # Sometimes serverless caches build artifacts that can confound testing
```

Comment:
Btw. these instructions will not work, since the DB is missing, right?

Reply:
The instructions assume (😬) that you have the correct database credentials set locally.

Reply:
I will update these instructions accordingly 👍